apache/metron
# METRON-2097 Install Metron MPack in Ambari 2.7.3.0

This PR has been submitted against the `METRON-2088-support-hdp-3.1` feature branch in support of running Metron on an HDP 3.1 cluster.

## Description

This change allows the Metron MPack to work with Ambari 2.7.3.0, which is the version of Ambari distributed in HDP 3.1.0.  

The change is also backwards compatible with Ambari 2.6.1.0, which is the version of Ambari that we currently install against for HDP 2.6.5.

The legacy CentOS 6 image is configured to deploy using Ambari 2.6.1.0 and HDP 2.6.5.

The new CentOS 7 image is configured to deploy using Ambari 2.7.3.0 and HDP 3.1.0

## Testing

1. Launch the CentOS 7 development environment. 
    * Login to http://node1:8080 with admin/admin and validate that it is Ambari 2.7.3.0
    * Ensure alerts are visible in the Alerts UI
    * Ensure that the Metron Service Check completes successfully.
    ```
   cd metron-deployment/development/centos7
   vagrant up
   ```

    ![CentOS 7 on Ambari 2 7 3 0](https://user-images.githubusercontent.com/2475409/56991158-01592880-6b65-11e9-939d-7654e5bfa9fa.png)

    ![HDP 3 1 0 0](https://user-images.githubusercontent.com/2475409/56991177-03bb8280-6b65-11e9-97b9-cee7f92a75eb.png)


1. Launch the CentOS 6 development environment.  
    * Login to http://node1:8080 with admin/admin and validate that it is Ambari 2.6.1.0
    * Ensure alerts are visible in the Alerts UI
    * Ensure that the Metron Service Check completes successfully.
    ```
   cd metron-deployment/development/centos6
   vagrant up
   ```

    ![CentOS 6 on Ambari 2 6 1 0](https://user-images.githubusercontent.com/2475409/56990693-c5719380-6b63-11e9-9848-b7d9118cbe70.png)

    ![HDP 2 6 5 0](https://user-images.githubusercontent.com/2475409/56990702-c99db100-6b63-11e9-9471-43ecc3dbc41a.png)



## Pull Request Checklist
